packages/components/Menu/src/MenuCallout/MenuCallout.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Animated, Modal, Platform, TouchableWithoutFeedback, View, StyleSheet, ScrollView } from 'react-native';
+
+import { Callout } from '@fluentui-react-native/callout';
+import { stagedComponent } from '@fluentui-react-native/framework';
+import { mergeProps } from '@fluentui-react-native/framework';
+
+import type { MenuCalloutProps } from './MenuCallout.types';
+import { menuCalloutName } from './MenuCallout.types';
+import { useMenuContext } from '../context';
+
+const AnimatedScrollView = Animated.createAnimatedComponent(ScrollView);
+
+export const MenuCallout = stagedComponent((props: MenuCalloutProps) => {
+  const context = useMenuContext();
+
+  return (_rest: MenuCalloutProps, children: React.ReactNode) => {
+    const mergedProps = mergeProps(props, _rest);
+    const tokens = props.tokens;
+
+    return Platform.OS === 'android' ? (
```

Comment:
Please separate the component into .tsx and .android.tsx so that the code is cleaner and easier to read

Reply:
Separated out to platform specific files.